Project.toml

```diff
@@ -1,22 +1,27 @@
 name = "LegolasFlux"
 uuid = "eb5f792d-d1b1-4535-bae3-d5649ec7daa4"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
non-breaking; all existing usage of LegolasFlux will work, but if you switch to the new `weights` / `loadweights!` it'll even work correctly with `BatchNorm`s 😄